wp-content/plugins/core/src/Object_Meta/Object_Meta_Definer.php

```diff
@@ -30,6 +32,13 @@ public function define(): array {
 					'settings_pages' => [ $container->get( Settings\General::class )->get_slug() ],
 				] );
 			},
+
+			// add appearance settings to pages.
+			Page_Appearance::class                              => static function ( ContainerInterface $container ) {
```

Comment:
I think this `=>` is also off?

Reply:
I thought this was odd-looking when I saw it, but I figured it _could_ be that way for a reason so I kept it. It shouldn't mess anything up so I've fixed the alignment for now.